CodeAnalysis/Xamarin.Interactive.CodeAnalysis/CodeAnalysis/Resolving/NativeDependencyResolver.cs

```diff
@@ -31,8 +31,7 @@ internal NativeDependencyResolver (TargetCompilationConfiguration compilationCon
         {
             CompilationConfiguration = compilationConfiguration;
 
-            agentArchitecture = Environment.Is64BitProcess &&
-                (HostEnvironment.OS != HostOS.Windows || compilationConfiguration.Sdk.IsNot (SdkId.ConsoleNetCore))
+            agentArchitecture = Environment.Is64BitProcess
```

Comment:
This is funny... why is `agentArchitecture` being decided based on the host process? Shouldn't any arch checking just check `CompilationConfiguration.Runtime.Architecture`?

It might also just be best to not touch `NativeDependencyResolver` in this PR at all since that's what @bojanrajkovic is working on.

Reply:
I was just removing the checks that were in place because of .NET Core being 32-bit on 64-bit Windows.

But yeah, I expect Bojan is deleting all of this. :-)